### ISSUE TITLE
Won't work in homekit app? 

### ISSUE THREAD

**Issue by Chewy48** [steve228uk/homebridge-magichome#69]
Hey everyone,

I'm pretty new to homebridge/ hoobs and have gotten 4 devices up and running, however I'm having troubled getting my magic home 2 to work? Would appreciate some help! So here's the info

Product I ordered: https://www.amazon.com/dp/B07C1LN7FZ/ref=cm_sw_r_cp_api_i_MKnWEb36JXE0Q

So this is the hoobs plugin I'm using. Its magic home 2

https://photos.app.goo.gl/SWRKRSkGbM2b4ct47

This is what my code looks like

https://photos.app.goo.gl/w9vVY31dzFwK3ncq9

I changed the ip to the device ip that i got from the magic home app as you can see here.

https://photos.app.goo.gl/2fBko6WmWKYkhMeZ6

It’s in my home app but it doesnt work at all. 

https://photos.app.goo.gl/LiR5mTnJisdkVPfe9

https://photos.app.goo.gl/NbuU7eMQXnnD488D9

https://photos.app.goo.gl/dAZsQrkuFePSMftN9

I'm kinda stumped on why it's not working.. If anyone can help me I'd greatly appreciate it!! Kind of a noob when it comes to this json stuff so take it easy on me haha.

**Comment by Khazanedar** [comment]
Did you ever get this fixed? I'm having similar issues. I added it to the Home app but it just says "No Response"

**Comment by Chewy48** [comment]
> Did you ever get this fixed? I'm having similar issues. I added it to the Home app but it just says "No Response"

Hey man, no haven't gotten it to work yet. Have you tried other magichome plugins? I really want it to work lol

**Comment by Khazanedar** [comment]
I tried homebridge-magichome-platform but I wasnt sure how to set it up. Kept getting errors when I saved the config.json file. This is driving me crazy, ive spent 7 hours on this just since last night lol

**Comment by Chewy48** [comment]
> I tried homebridge-magichome-platform but I wasnt sure how to set it up. Kept getting errors when I saved the config.json file. This is driving me crazy, ive spent 7 hours on this just since last night lol

Dude same here! I can't get the json right I'm that one it always says error when I paste it in. Tried changing things around but I don't know json hhah driving me crazy too! I'm sure it's something simple but I can't find any me who knows what they're doing lol

**Comment by Khazanedar** [comment]
I was so stoked when I found these plugins because I didn't think they existed, I had settled with just using IFTTT webhooks to trigger the LED strips. But now I'm disappointed that it isnt working lol

**Comment by Chewy48** [comment]
> I was so stoked when I found these plugins because I didn't think they existed, I had settled with just using IFTTT webhooks to trigger the LED strips. But now I'm disappointed that it isnt working lol

Same man!! I bet the other plugin will work if we can get the json right but I can never get it right

**Comment by Khazanedar** [comment]
Yep, the json confuses me alot because i feel like it messes with the "name": "Config" and port and stuff. doing it under "accessories" wasnt as confusing. Also the example json has sooooo many different things and its tough to delete what you dont want. Also if you dont have multiple lights i dont think it works...?
